commands/build.js

```diff
@@ -110,6 +110,8 @@ module.exports = {
             if (hasXrType && hasStartUrl) {
               xrType = j.xr_type;
               xrDetails = j.xr_details;
+              physicsUrl = j.physics_url;
```

Comment:
This isn't declared so it's a strict mode error.

Reply:
b35d109